ProjectShop/order/views.py

```diff
@@ -1,10 +1,19 @@
-from rest_framework import viewsets
+from rest_framework import viewsets, renderers
+from rest_framework.decorators import action
+from rest_framework.response import Response
 
-from order.models import OrderModel
-from order.serializers import OrderDetailSerializer
+from order.models import Order
+from order.models import OrderItems
 
+from order.serializers import OrderDetailSerializer, OrderItemsSerializer
 
 class OrderViewSet(viewsets.ModelViewSet):
     """This is viewset for order model"""
-    queryset = OrderModel.objects.all()
+    queryset = Order.objects.all()
     serializer_class = OrderDetailSerializer
+    http_method_names = ['post', 'get', 'patch', 'put']
+
+class OrderItemsViewSet(viewsets.ReadOnlyModelViewSet):
```

Comment:
```suggestion
class OrderItemsViewSet(viewsets.ModelViewSet):
```
тоді зможемо додавати, видаляти і редагувати ордерайтемс
чи то так задумано щоб заблокувати ці методи?

Reply:
> тоді зможемо додавати, видаляти і редагувати ордерайтемс
> чи то так задумано щоб заблокувати ці методи?

Хм, доречі тут трішки не додумав, тому що в клієнта не буде можливості добавляти позиції ніяк, окрім адмінки, треба змінити